File: sentimental-mario-less/mario.py

# equivelant of do while in python
height = 0
while height > 8 or height < 1:
    try:
        height = int(input("Height: "))
    except ValueError:
        height = int(input("Height: "))


# looping through height then doing the spacing and then printing the hash and new line
for i in range(height):
    for space in range(height - i - 1):
        print(" ", end="")
    for j in range(i + 1):
        print("#", end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int height;
int main(void)
{
    do
    {
        height = get_int("Please input height: ");
    }
    while(height < 1 || height > 8);


    for(int i = 0; i < height; i++)
    {
        for(int space = 0; space < height - i - 1; space++)
        {
            printf(" ");
        }
        for(int j = 0; j <= i; j++)
        {
            printf("#");
        }
        for(int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }




}